federation/develop/develop.sh

```diff
@@ -123,7 +115,9 @@ function get_version() {
     kube_version="${KUBE_VERSION}"
   else
     # Read the version back from the versions file if no version is given.
-    kube_version="$(jq -r '.KUBE_VERSION' ${VERSIONS_FILE})"
+    kube_version="$(cat ${VERSIONS_FILE} | python -c '\
```

Comment:
cc @ixdy. 

This was a page taken straight out of @ixdy's book - https://github.com/kubernetes/release/pull/207